l10n_it_website_sale_corrispettivi/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,14 @@
+**Italiano**
+
+Il comportamento dipende da come è configurato il partner associato all'utente loggato, ad esempio:
+
+* Se il partner associato all'utente loggato ha il flag *Usa corrispettivi* abilitato, la spunta *Voglio fattura* sarà disabilitata;
+* Se il partner associato all'utente loggato ha il flag *Usa corrispettivi* disabilitato, la spunta *Voglio fattura* sarà abilitata;
+
+**English**
+
+The behavior depends on how the partner associated with the logged in user is configured, for example:
+
+* If the partner associated with the logged in user has the *Use corrispettivi* flag enabled, the check *Want invoice* will be disabled;
+* If the partner associated with the logged in user has the *Use corrispettivi* flag disabled, the check *Want invoice* will be enabled;
```

Comment:
corrispettivi → sums due

Anche qui verifica nelle altre parti del readme.

Reply:
> corrispettivi → sums due

In base a quanto discusso in
[#977 (comment)](https://github.com/OCA/l10n-italy/pull/977#discussion_r255326522)

Corrispettivi → Ricevute/Receipts

Sia nel readme che nelle stringhe sorgenti di traduzione.